.github/workflows/ai-review.yaml

```
@@ -0,0 +1,78 @@
+name: Custom AI Code Review
+description: "Automated code quality and documentation review on pull requests"
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)